api/graphql/mutations/role.js

```diff
@@ -67,20 +61,25 @@ export async function addRoleToMember ({userId, groupRoleId, personId, groupId})
   }
 }
 
-export async function removeRoleFromMember ({userId, memberRoleId, personId, groupId}){
+export async function removeRoleFromMember ({userId, groupRoleId, personId, groupId}){
   if (!userId) throw new GraphQLYogaError(`No userId passed into function`)
 
-  if (personId && memberRoleId && groupId) {
+  if (personId && groupRoleId && groupId) {
     const groupMembership = await GroupMembership.forIds(userId, groupId).fetch()
```

Comment:
oh is userId the one assigning the badge? then i think it should be called creatorId, or moderatorId